doc/man3/OPENSSL_malloc.pod

```diff
@@ -60,6 +62,9 @@ CRYPTO_mem_leaks, CRYPTO_mem_leaks_fp - Memory allocation functions
 
  int CRYPTO_set_mem_debug(int onoff)
 
+ env
+ The env var OPENSSL_MALLOC_FAILURES controls how often malloc/realloc
```

Comment:
Finish that sentence, please? Also, do you really mean to have that description in the middle of the synopsis?